modules/api_gateway/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "invoke_url" {
+  description = "Base url used to invoke this module's api endpoints"
+  value       = aws_api_gateway_deployment.respond.invoke_url
+}
+
+output "slack_listener_api_endpoint_arn" {
+  description = "ARN of the slack listener API endpoint"
+  value       = "${aws_api_gateway_rest_api.api.execution_arn}/*/${aws_api_gateway_method.event_listener_post.http_method}${aws_api_gateway_resource.event_listener.path}"
```

Comment:
This doesn't seem like it's outputting an ARN to me. Don't think I have seen an ARN with astericks in it before. What's this supposed to look like?

Reply:
here's little more context for this: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-invoke-api.html